example-app/SantokuApp/src/features/terms/components/__snapshots__/TermsAgreementOverlayComponent.test.tsx.snap

```diff
@@ -162,12 +162,9 @@ Array [
               injectedJavaScriptBeforeContentLoadedForMainFrameOnly={true}
               injectedJavaScriptForMainFrameOnly={true}
               javaScriptEnabled={true}
-              messagingEnabled={false}
+              messagingEnabled={true}
```

Comment:
Why the change?

Reply:
Perhaps this [pr](https://github.com/react-native-webview/react-native-webview/commit/6b8e4f3f5f3cc006ebca8cb3c7ab692805824685) has changed things.

I feel it will always be `true`.
https://github.com/react-native-webview/react-native-webview/commit/6b8e4f3f5f3cc006ebca8cb3c7ab692805824685#diff-52ca22dffa10ef8f44aba53190c09e6ee08c9b57407d4fbd25403d49f9b800c1R190

The `messagingEnabled` is not a publicly exposed API, but is used internally by react-native-webview.
The messageHandler is always registered, but if onMessage is not specified, it will still do nothing, so it does not seem to have much of an effect on the operation.